### PR TITLE
[ON HOLD] [BackgroundsV2 Project] [NON-MODULAR] [NO GBP] Scaling Paycheck Code, Sets Paychecks to Every 20 Minutes By Default

### PR DESCRIPTION
**Pull request by RimiNosha**
## About The Pull Request

So, paychecks happen every 5 minutes. This would be fine if it weren't for the fact that I'm attaching more content onto the HoP, and it's frequency is based off of paychecks. This is not gonna end well if I leave it at 5 minutes.

So, I just increased the value, and was like "I bet it just gives a constant amount each payout", which was correct, so I adjusted the number, and then I was like "Ugh, someone's probably gonna change this later, I'll just make it so they only need to adjust the wait variable", and so this set of edits was born!

## How This Contributes To The Skyrat Roleplay Experience
 HoP not drowning in mail from centcom when that gets added is nice, no?

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  When I get VV to play nice
</details>

### CODE REVIEW

**Comment by RimiNosha** [comment]
FUCK, I FORGOT A MINUTE ISN'T ONE AAAAAAA

**Comment by RimiNosha** [comment]
WHY DID IT PAY MORE?!

Fuck. I need help from a maths person, I can't do this.

File: code/modules/economy/account.dm [hunk 1]
```diff
@@ -152,12 +152,14 @@
  * * amount_of_paychecks - literally the number of salaries, 1 for issuing one salary, 5 for issuing five salaries.
  * * free - issuance of free funds, if TRUE then takes funds from the void, if FALSE (default) tries to send from the department's account.
  */
-/datum/bank_account/proc/payday(amount_of_paychecks, free = FALSE)
+/datum/bank_account/proc/payday(amount_of_paychecks, free = FALSE, initial_paycheck = FALSE) // SKYRAT EDIT - BACKGROUNDS - Proper scaling paychecks - ORIGINAL: /datum/bank_account/proc/payday(amount_of_paychecks, free = FALSE)
 	if(!account_job)
 		return
-	var/money_to_transfer = round(account_job.paycheck * payday_modifier * amount_of_paychecks)
+	var/money_to_transfer = round((initial_paycheck ? (account_job.paycheck * payday_modifier) : clamp(account_job.paycheck * payday_modifier, 0, PAYCHECK_CREW)) * amount_of_paychecks * background_multiplier) // SKYRAT EDIT - BACKGROUNDS - Proper scaling paychecks - ORIGINAL:	var/money_to_transfer = round(account_job.paycheck * payday_modifier * amount_of_paychecks)
```
Comment (nevimer) [inline code/modules/economy/account.dm:158]: i'd yeet the payday_modifier var wherever possible imo.

**Comment by RimiNosha** [comment]
On hold while I focus my attention elsewhere. No, this is not dead. This is not going anywhere, especially since I've spent so damn much time figuring out details for the background project.

**Pull request closed without merging**